.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug Report
+about: 'Create a report to help us improve. '
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- 
+Read the guidelines for filing an issue first. 
+
+https://github.com/microsoft/python-language-server/blob/master/TROUBLESHOOTING.md#filing-an-issue
+-->
+
+## Environment data
+- Language Server version: XXX   
+- OS and version: XXX
+- Python version (& distribution if applicable, e.g. Anaconda): XXX
+
+## Expected behaviour
+
+XXX
+
+## Actual behaviour
+
+XXX
+
+
+## Logs
+
+```
+XXX
+```
+
+## Code Snippet / Additional lnformation
+
+```
```

Comment:
Converting this to:

    ```python
    # 
    ```

Would improve what gets pasted.

Reply:
sounds good

Reply:
Agh, commented in the wrong place. Resolving.